sources/vcdMerge/src/vcdMerge.cpp

```diff
@@ -46,19 +46,31 @@ int main(int argc, const char *argv[])
     try
     {
         // Parse input parameters
-        CLI::CliMerge cli;
-        cli.Parse(argc, argv);
+        CLI::CliMerge *cli;
+        try
+        {
+            cli = new CLI::CliMerge();
```

Comment:
Where is `delete`? You can use `make_unique` here.

Reply:
Also: why is this needed at all? Ctor should not throw.

Reply:
Tclap can throw an exception there during the construction of the object to parse the input line. The validation list for some parameters cannot be empty.